tests/expected/reach/bounds/unreachable/expected

```diff
@@ -0,0 +1 @@
+UNREACHABLE
```

Comment:
This only guarantees that there is at least one check that is unreachable, not that all expected checks are